src/app/components/users-panel/users-panel.component.html

```diff
@@ -103,12 +106,12 @@ <h6 id="property-title">Saved Location</h6>
             Map link not saved.
           </div>
         </ng-template>
-
-        <div style="display: flex; flex-direction: column; justify-content: center">
-          <button class="button hollow" (click)=openExportProjectModal()>
-            <i class="fas fa-save"></i>  Save map link
-          </button>
-        </div>
+        <!-- <div *ngIf="!activeProject.system_id || (activeProject.system_id && !activeProject.system_id.includes('project'))" 
```

Comment:
should we just drop?  or add a link to teh jira ticket here? then we could drop whenever we are certain that we aren't going to add this feature in.  (same applies fo `openExportProjectModal` method)

Reply:
I'll drop it for now.
Created https://jira.tacc.utexas.edu/browse/DES-2156.